documentation/docs/iv-viewer/api.md

```diff
@@ -0,0 +1,267 @@
+```js
+viewer = viewer.destroy();
+```
+
+### API (FullScreenViewer)
+FullScreenViewer is extended from ImageViewer. So it shares the same ImageViewer api along with some FullScreenViewer API.
+
+Creating an instance
+
+```js
+import { FullScreenViewer } from 'iv-viewer';
+
+const viewer = new FullScreenViewer(options);
+```
+Unlike ImageViewer you don't have to pass container for the viewer as it will be initialized in pre-defined full screen container.
+
+First argument is to provide configuration options for the FullScreenViewer. This argument is optional.
+
+#### Instance methods (FullScreenViewer)
+FullScreenViewer inherits all the instance method of ImageViewer. In additional to that it has following methods.
+
+**show(imgSrc, highResImg)**
+
+Show the full screen viewer with passed image on the show method. You can pass two different resolution of the image as first and second argument (optional). See why do you need it at [better image loading](#better-image-loading) section. 
+
+```js
+viewer.show('image.png', 'hi-res-image.png');
+```
+
+**hide()**
+
+Hide/Close the fullScreen viewer. 
+```js
+viewer.hide();
+```
+
+### Better image loading
+To improve the perceived experience, it is always recommended to show the already loaded image or the low quality image on the viewer and let the ImageViewer load high quality image in parallel. 
+
+It will also try to preview the high quality image while it's loading so it will give a progressive loading effect.
+
+ImageViewer provides api to do this. 
+Container mode
+```js
+viewer.load('image.png', 'hi-res-image.png');
+```
+
+FullScreen mode
+```js
+viewer.show('image.png', 'hi-res-image.png');
+```
+
+Image Mode
+```html
+<img id="image" src="image.png" data-high-res-src="hi-res-image.png" />
+```
+
+In all of the above example it will first try to display the first image and then load the second image (if passed) in parallel.
+
+The second image is optional, which you should pass when you feel you can improve the image loading experience by first showing low quality image and then progressively update it with high quality image.
+
+### Demo
+codesandbox preview link: [https://8ypwzryom0.codesandbox.io/](https://8ypwzryom0.codesandbox.io/)
+
+codesandbox link: [https://codesandbox.io/s/8ypwzryom0](https://codesandbox.io/s/8ypwzryom0)
+
+### Like this
+[:star: this repo](https://github.com/s-yadav/iv-viewer)
+
+### Major updates
+#### v2.1.0
+
+- It's a complete rewrite of ImageViewer with no jQuery requirement in ES6.
+- While the options and instance method are same the way you use a ImageViewer and FullScreenView is changed. The v1 API will no longer be supported.
+- Published on the npm. V1 was not available on npm.
+- Changed the package name to iv-viewer. image-viewer name was not available on npm. 
+- Added some listeners
+- Added zoom in/out buttons
+- Some SCSS variable for easier overrides.
+
+## Contributors ✨
```

Comment:
**Suggestion**
Can remove Contributors section from here, as its already part of entry file.